packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -224,15 +225,15 @@ const MultiSelect = React.forwardRef(function MultiSelect(
 
   return (
     <div className={wrapperClasses}>
-      <label className={titleClasses} {...getLabelProps()}>
+      <Text as="label" className={titleClasses} {...getLabelProps()}>
         {titleText && titleText}
```

Comment:
It may be that we want the outer `<label>` and then wrap `titleText` in the `Text` component, if `titleText` is not `PropTypes.node`